hub/demo/src/components/CompetitionLeaderboardTable.tsx

```diff
@@ -0,0 +1,297 @@
+          ) : (
+            <Text>No entries submitted yet for this competition.</Text>
+          )}
+        </>
+      ) : (
+        <Table.Root {...tableProps}>
+          {sorted && (
+            <Table.Head>
+              <Table.Row>
+                <Table.HeadCell
+                  column="model"
+                  sortable
+                  style={{ minWidth: '16rem' }}
+                >
+                  Model
+                </Table.HeadCell>
+
+                <Table.HeadCell
+                  column="namespace"
+                  sortable
+                  style={{ minWidth: '12rem' }}
+                >
+                  Author
+                </Table.HeadCell>
+
+                <Table.HeadCell style={{ minWidth: '12rem' }}>
+                  Entry Status
+                </Table.HeadCell>
+
+                {selectedBenchmarkColumns.map((column) => (
+                  <Table.HeadCell column={column} sortable key={column}>
+                    <Flex
+                      as="span"
+                      direction="column"
+                      style={{ textTransform: 'none' }}
+                    >
+                      {column.includes('/') && (
+                        <Text
+                          as="span"
+                          size="text-2xs"
+                          color="current"
+                          style={{
+                            marginBottom: '-0.1rem',
+                            display: 'inline-block',
+                          }}
+                        >
+                          {column.split('/').slice(0, -1).join('/')}/
+                        </Text>
+                      )}
+                      <Text
+                        as="span"
+                        size="text-s"
+                        weight={600}
+                        color="current"
+                      >
+                        {column.split('/').at(-1)}
+                      </Text>
+                    </Flex>
+                  </Table.HeadCell>
+                ))}
+              </Table.Row>
+            </Table.Head>
+          )}
+
+          <Table.Body>
+            {!sorted && <Table.PlaceholderRows />}
+
+            {sorted?.map((evaluation, index) => (
+              <Table.Row key={index}>
+                {evaluation.provider === 'local' ? (
+                  <Table.Cell
+                    href={`/models/${evaluation.namespace}/${evaluation.model}/${evaluation.version || 'latest'}`}
+                  >
+                    <Text size="text-s" weight={600} color="sand-12">
+                      {evaluation.model}
+                    </Text>
+                  </Table.Cell>
+                ) : (
+                  <Table.Cell>
+                    <Text size="text-s" weight={600} color="sand-12">
+                      {evaluation.model}
+                    </Text>
+                  </Table.Cell>
+                )}
```

Comment:
@arktoswb I wasn't sure if we can expect `provider: "local"` entry rows to have their models uploaded to the registry so that we can link to it via `/models/${evaluation.namespace}/${evaluation.model}/${evaluation.version || 'latest'}`. Is this correct or do I need to remove the link?

Reply:
It appears to be correct

Reply:
For provider=`local` we will want to render `${evaluation.namespace}/${evaluation.model}/${evaluation.version}` similar to how we render agents